rules/common_settings.bzl

```diff
@@ -123,6 +123,24 @@ string_list_flag = rule(
     doc = "A string list-typed build setting that can be set on the command line",
 )
 
+def _flag_impl(ctx):
+    return BuildSettingInfo(value = ctx.build_setting_value)
+
+repeatable_string_flag = rule(
+    implementation = _flag_impl,
+    build_setting = config.string_list(
+        flag = True,
+        repeatable = True,
+    ),
+    attrs = {
+        "scope": attr.string(
+            doc = "The scope indicates where a flag can propagate to",
+            default = "universal",
+        ),
+    },
+    doc = "A string-typed build setting that can be set on the command line. Multiple settings do not overwrite each other; they are concatenated into a list",
```

Comment:
Please clarify in the docs here and for `string_list_flag` how the two rules differ. For example,

* for `string_list_flag` - "A string list-typed build setting which expects its value on the command line to be given in comma-separated format; for example, `--//my/setting=foo,bar` will be parsed as `["foo", "bar"]`. Contrast with `repeatable_string_flag`.
* for `repeastable_string_flag` - "A string-typed build setting which can be specified multiple times on the command line, with the values concatenated into a single list; for example, `--//my/setting=foo --//my/setting=bar` will be parsed as `["foo", "bar"]`. Contrast with `string_list_flag`.

Reply:
Great doc suggestions!